notifications_utils/template.py

```diff
@@ -1,5 +1,6 @@
 import re
 import math
+import bleach
```

Comment:
Discussed IRL, bleach only strips out complete HTML, it doesn’t handle breaking out of attributes, eg:
```
https://example.com"onclick="alert('hi')
```